lib/index.ts

```diff
@@ -673,7 +673,9 @@ export default class Backburner {
 
   private _runExpiredTimers() {
     this._timerTimeoutId = null;
-    this.run(this, this._scheduleExpiredTimers);
+    this.begin();
+    this._scheduleExpiredTimers();
```

Comment:
run has a try/finally, i believe we have lost that now. should we bring it back?